### PR TITLE
Merchant invoices user story 2

### LINKED ISSUE #26: Merchant Invoice Show Page
As a merchant
When I visit my merchant's invoice show page(/merchants/merchant_id/invoices/invoice_id)
Then I see information related to that invoice including:
- Invoice id
- Invoice status
- Invoice created_at date in the format "Monday, July 18, 2019"
- Customer first and last name

### PR DESCRIPTION
**Pull request by JCNapier**
I finally completed the first show page user story. 

1. Changed the test set up to let blocks

2. Added Status to FactoryBot for invoices, and invoice items.

3. Changed migration to "Add Column" instead of "Change Column"

4. Removed Validation for credit Card Number, in order to seed database. 

5. Completed Show Page user story one with testing and functionality.

### CODE REVIEW

File: spec/features/merchants/invoices/index_spec.rb [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
-    visit "/merchants/#{@merchant.id}/invoices"
-  end
 
-  it 'shows each merchant invoice id' do
-    expect(page).to have_content(@invoice_item.invoice_id)
-    expect(page).to have_content(@invoice_item2.invoice_id)
-    expect(page).to have_content(@invoice_item3.invoice_id)
+  let!(:merchant_1) {FactoryBot.create(:merchant)}
+
+  let!(:item_1) {FactoryBot.create(:item, merchant: merchant_1)}
```
Comment (joel-grant) [inline spec/features/merchants/invoices/index_spec.rb:7]: This is so cool, I hadn't thought of using let AND factorybot.

**Review by joel-grant — APPROVED** [review]

**Comment by kbhoffmann** [comment]
I see there is a small merge conflict.  John, do you know what is supposed to stay in the merge conflict?  Would you mind resolving the conflict if you know what its supposed to be?

**Comment by JCNapier** [comment]
Totally happy to resolve the merge conflict

**Comment by JCNapier** [comment]
Don't merge this PR yet I am now having issues with something I wasnt before and I need to figure out why.

**Pull request closed without merging**